apps/admin-gui/src/app/shared/components/services-list/services-list.component.html

```diff
@@ -85,13 +85,13 @@
         [pageSizeOptions]="pageSizeOptions">
       </mat-paginator>
     </div>
-
   </div>
 </div>
 
 <mat-spinner *ngIf="exporting" class="ml-auto mr-auto"></mat-spinner>
 
-<app-alert *ngIf="dataSource.filteredData.length === 0 && (this.services !== undefined && this.services.length !== 0)" color="warn">
+<app-alert *ngIf="dataSource.filteredData.length === 0 && (this.services !== undefined && this.services.length !== 0)"
+           color="warn">
   {{'SHARED_LIB.UI.ALERTS.NO_FILTER_RESULTS_ALERT' | translate}}
 </app-alert>
 
```

Comment:
So now the message that there are no services is not showing properly(row 98-100). Please rewrite it with:
`<app-alert *ngIf="this.services === undefined || this.services.length === 0" color="warn">
  {{'SHARED.COMPONENTS.SERVICES_LIST.NO_SERVICES_WARNING' | translate}}
</app-alert>`